src/sentry/tasks/statistical_detectors.py

```diff
@@ -924,16 +924,14 @@ def query_functions_timeseries(
     agg_function: str,
 ) -> Generator[tuple[int, int | str, SnubaTSResult], None, None]:
     projects = [project for project, _ in functions_list]
-    project_ids = [project.id for project in projects]
 
     # take the last 14 days as our window
     end = start.replace(minute=0, second=0, microsecond=0) + timedelta(hours=1)
-    params: ParamsType = {
-        "start": end - timedelta(days=14),
-        "end": end,
-        "project_id": project_ids,
-        "project_objects": projects,
-    }
+    params = SnubaParams(
+        start=end - timedelta(days=14),
+        end=end,
+        projects=projects,
+    )
```

Comment:
Didn't use `SnubaParams(stats_period=` here since the existing functionality sets end to 1hr in the future